pkgs/by-name/fn/fna3d/package.nix

```
@@ -0,0 +1,37 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, SDL2
+}:
+stdenv.mkDerivation rec {
+  pname = "fna3d";
+  version = "23.11";
+
+  src = fetchFromGitHub {
+    owner = "FNA-XNA";
+    repo = "FNA3D";
+    rev = version;
+    fetchSubmodules = true;
+    hash = "sha256-UXphZQrk+MUOcUIqZ0ho3HTP5vNZjN9ifsANOJbTfVE=";
+  };
+
+  buildInputs = [ cmake SDL2 ];
```

Dependencies that need to be executed at build time have to go in `nativeBuildInputs`

```suggestion
  nativeBuildInputs = [ cmake ];
  
  buildInputs = [ SDL2 ];
```